MekHQ/src/mekhq/campaign/io/Migration/ForceIconMigrator.java

```diff
@@ -27,19 +28,206 @@
+                            default:
+                                break;
+                        }
+                        break;
+                    case "Inner Sphere/":
+                        switch (piece.getFilename()) {
+                            case "Federated Suns.png.png":
+                                piece.setFilename("Federated Suns.png");
+                                break;
+                            case "Rim Worlds Republic.png":
+                                piece.setCategory("Periphery/");
+                                break;
+                            default:
+                                break;
+                        }
+                        break;
+                    default:
+                        break;
+                }
+            }
+        }
+
+
+        if (icon.getPieces().containsKey(LayeredForceIconLayer.TYPE)) {
+            boolean hasLAM = false;
+
+            for (final ForcePieceIcon piece : icon.getPieces().get(LayeredForceIconLayer.TYPE)) {
+                switch (piece.getCategory()) {
+                    case "NATO/":
+                        if ("Vehicle (Wheeled Tracked).png".equals(piece.getFilename())) {
+                            icon.setFilename("Vehicle (Tracked Wheeled).png");
+                        }
+                        break;
+                    case "StratOps/":
+                        switch (piece.getFilename()) {
+                            case "LAM.png":
+                                hasLAM = true;
+                                break;
+                            case "Vehicle (Hover Wheeled Armoured).png":
+                                icon.setCategory("Pieces/Types/NATO/");
+                                icon.setFilename("Vehicle (Mixed).png");
+                                break;
+                            default:
+                                break;
+                        }
+                        break;
+                    default:
+                        break;
+                }
+            }
+
+            if (hasLAM) {
+                icon.getPieces().get(LayeredForceIconLayer.TYPE).removeIf(piece ->
+                        "StratOps/".equals(piece.getCategory()) && "LAM.png".equals(piece.getFilename()));
+                icon.getPieces().get(LayeredForceIconLayer.TYPE)
+                        .add(new ForcePieceIcon(LayeredForceIconLayer.TYPE, "StratOps/", "BattleMech (Left).png"));
+                icon.getPieces().get(LayeredForceIconLayer.TYPE)
+                        .add(new ForcePieceIcon(LayeredForceIconLayer.TYPE, "StratOps/", "Aerospace (Right).png"));
+            }
+        }
+
+        return icon;
+    }
+
+    private static StandardForceIcon migrateStandardForceIcon0497(final StandardForceIcon icon) {
+        switch (icon.getCategory()) {
+            case "Pieces/Alphanumerics/StratOps/":
+                if ("C3\302\240.png".equals(icon.getFilename())) {
```

Comment:
Are the backslashes here intentional?

Reply:
Yeah, the file had hidden characters in the name.